javascript/javascript1/week2/formal_fullname.js

```diff
@@ -0,0 +1,13 @@
+const namethatIwant = getFullname('Benjamin', 'Hughes', true);
```

Comment:
It is better to call the function after you declare it, not before. Move this line below the function